libp2p/exceptions.py

```diff
@@ -1,6 +1,8 @@
-class ValidationError(Exception):
+class BaseLibp2pError(Exception):
+    pass
+
+
+class ValidationError(BaseLibp2pError):
     """
     Raised when something does not pass a validation check.
     """
```

Comment:
```
"""
pass
```

Reply:
pass is not necessary since we have the docstring.